apps/web/src/hooks/useIsOfficialHost.ts

```diff
@@ -1,9 +1,39 @@
 import { useMemo } from 'react'
-import { IS_OFFICIAL_HOST, OFFICIAL_HOSTS } from '@/config/constants'
+import { IPFS_HOStS, IS_OFFICIAL_HOST, OFFICIAL_HOSTS } from '@/config/constants'
+import { version } from '../../../../package.json'
+import useAsync from './useAsync'
+
+const GITHUB_API_URL = 'https://api.github.com/repos/5afe/safe-wallet-ipfs/releases/tags/'
+
+async function getGithubRelease(version: string) {
+  const resp = await fetch(`${GITHUB_API_URL}/v${version}`, {
+    headers: {
+      Accept: 'application/vnd.github.v3+json',
+    },
+  })
+  if (!resp.ok) return false
+  return await resp.json()
+}
```

Comment:
You could even try to compare the CID requested with the latest CID (https://api.github.com/repos/5afe/safe-wallet-ipfs/releases/latest) and suggest/redirect the user to the latest version

Reply:
Let's leave it for another PR. This one aims to just toggle the logo and footer on/off depending on the legitimacy of the current host.